docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -82,6 +82,50 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 > [!TIP]
 > For more information and troubleshooting tips on .NET Core debugging with OmniSharp in Visual Studio Code, see [Instructions for setting up the .NET Core debugger](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md).
 
+## Add a Class
```

Comment:
Sentence case for headings

```suggestion
## Add a class
```

I can merge for you and add VB later.

Reply:
Yay! Thank you!